src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -1644,81 +1416,76 @@ triggers) a wall clock time expressed as an offset from cycle point:
-offsets are normally positive, to trigger some time *after* the wall-clock
-time is equal to task cycle point.
-
-Clock-triggers have no effect on scheduling if a workflow is running sufficiently
-far behind the clock (e.g. after a delay, or because it is processing archived
-historical data) that the trigger times, which are relative to task cycle
-point, have already passed.
+Here, ``foo[2025-08-23T00]`` would trigger (other dependencies allowing)
+when the wallclock time reaches ``2025-08-23T02``. Clock-trigger
+offsets are normally positive, to trigger *after* the wallclock time is equal
+to the task cycle point.
 
 
 .. _ClockExpireTasks:
 
 Clock-Expire Triggers
 ^^^^^^^^^^^^^^^^^^^^^
 
-Tasks can be configured to *expire* - i.e. to skip job submission and
-enter the *expired* state - if they are too far behind the wall clock when
-they become ready to run, and other tasks can trigger off this. As a possible
-use case, consider a cycling task that copies the latest of a set of files to
-overwrite the previous set: if the task is delayed by more than one cycle there
-may be no point in running it because the freshly copied files will just be
-overwritten immediately by the next task instance as the workflow catches back up
-to real time operation. Clock-expire tasks are configured with
+Tasks can be configured to *expire* and skip job submission if they are too far
+behind the wallclock when they become ready to run (and other tasks can trigger
+off of this).
+
+For example, if a clock-triggered task always copies the latest of a set of
+files to overwrite the previous set, in every cycle, there would be no point in
+running it late because the files it copied would be immediately overwritten by
+the next task instance as the workflow catches back up to real time operation.
+
+Clock-expire tasks are configured with
 :cylc:conf:`[scheduling][special tasks]clock-expire` using a syntax like
 :cylc:conf:`clock-trigger <[scheduling][special tasks]clock-trigger>`
-with a date-time offset relative to cycle point.
-The offset should be positive to make the task expire if the wall-clock time
-has gone beyond the cycle point. Triggering off an expired task typically
-requires suicide triggers to remove the workflow that runs if the task has not
-expired. Here a task called ``copy`` expires, and its downstream
-workflow is skipped, if it is more than one day behind the wall-clock:
+with a datetime offset relative to cycle point.
+The offset should be positive to make the task expire if the wallclock time
+has gone beyond the cycle point.
 
-.. code-block:: cylc
+.. warning::
 
-   [scheduler]
-      allow implicit tasks = True
-      cycle point format = %Y-%m-%dT%H
-   [scheduling]
-       initial cycle point = 2015-08-15T00
-       [[special tasks]]
-           clock-expire = copy(-P1D)
-       [[graph]]
-           P1D = """
-               model[-P1D] => model => copy => proc
-               copy:expired => !proc
-           """
```

Comment:
Why remove this example?

Reply:
Because we don't need this bit in Cylc 8: `copy:expired => !proc` and there's no point in giving an example with a specific graph just to show the `clock-expire` config line.

I suppose we could contrive another use for the `expired` trigger, but the obvious candidates would be better done with the event handler (`copy:expired => alert-me` ... )

Reply:
(Any ideas?)

Reply:
(Although we don't have an "expired" event!) ... https://github.com/cylc/cylc-flow/issues/4527